_posts/2020-07-12-ray-tune.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: post
+title: Dask-ML and Ray have the same features for model selection
+tagline: Modern hyperparameter optimizations, Scikit-learn support, framework support and scaling to many machines.
+author: <a href="https://stsievert.com">Scott Sievert</a> (University of Wisconsin–Madison)
+tags: [machine-learning, dask-ml, dask, ray]
+theme: twitter
+---
+
+*This post also lives at http://stsievert.com/2020/03/ (with better styling)*
```

Comment:
I assume you are planning on fixing this link once the editing is done?

Reply:
Yup, correct. I've added a TODO placeholder.